docs/deployment/containers/docker.txt

```diff
@@ -100,7 +100,7 @@ So, let's create a second node, called ``crate02``, and configure it to ping
     sh$ docker run -d --name=crate02 \
           --net=crate -p 4202:4200 --env CRATE_HEAP_SIZE=2g \
           crate -Cnetwork.host=_site_ \
-                -Cdiscovery.zen.ping.unicast.hosts=crate01
```

Comment:
agree with jordi

perhaps add something like this:

```
.. NOTE::

    In versions of CrateDB prior to 4.0, you must use ``discovery.zen.ping.unicast.hosts`` instead of ``discovery.seed_hosts``.